architecture/additional_concepts/authorization.adoc

```diff
@@ -399,10 +405,16 @@ Admission uses the following approach to create the final security context for
 the pod:
 
 . Retrieve all SCCs available for use.
-. Generate field values for any security context setting that was not specified
+. Generate field values for security context settings that were not specified
 on the request.
 . Validate the final settings against the available constraints.
 
+For a *FSGroup* SCC strategy of `MustRunAs`, if the pod spec defines a _fsGroup_ id then that id value must equal the default fSGroup id (first id in the first range in the namespace's "openshift.io/sa.scc.supplemental-groups" annotation). Otherwise, the pod is not authorized by that SCC and the next SCC is evaluated.
+If the *FSGroup* setting is `RunAsAny`, or if the pod spec omits a _fsGroup_ id then the pod matches the SCC based on FSGroup.
```

Comment:
I'd like to present some of the group id interactions (from that table that I emailed) relating admission, scc policies, namespace ranges, and ids defined in pods. Do you think that would be better in a different doc, or in  an appendix to this doc, or sprinkled in various sections of this doc? This is the reason I'm being more specific above...